golang_checks.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -euo pipefail
+
+export PATH="$GOPATH/bin:$PATH"
+RET=0
+
+echo "Installing gometalinter..."
+go get -u -t -v github.com/alecthomas/gometalinter
+
+echo "Starting code checks in parallel..."
+
+find . -type f -name '*.go' ! -path './vendor/*' | xargs -P 20 -n 20 gofmt -s -l > gofmt.log 2>&1 &
+GOFMT_PID="$!"
+
+# A deadline of 10m may seem excessive, but on slow machines it's preferable to allow it to take longer than to fail
+# the build.
+gometalinter --deadline=600s --config=./gometalinter.json --vendor ./... > metalinter-required.log 2>&1 &
+METALINTER_REQUIRED_PID="$!"
+
+gometalinter --deadline=90s --vendor --disable-all --enable=goconst --enable=errcheck --enable=dupl \
+    --enable=vet --enable=goimports ./... > metalinter-optional.log 2>&1 &
+METALINTER_OPTIONAL_PID="$!"
+
```

Comment:
Why are we using two separate invocations? Can't we run everything one pass?

Reply:
Good point. I will remove it in next version